keras2onnx/proto/__init__.py

```diff
@@ -35,3 +36,11 @@ def _check_onnx_version():
     except ImportError:
         is_tf_keras = True
         from tensorflow.python import keras
+
+
+def is_keras_older_than(version_str):
```

Comment:
Shall we cover the case "is_keras_older_than_or_equal_to"

Reply:
can be achieved by not _later_than, don't dup it.